contracts/Platform.sol

```diff
@@ -27,6 +25,30 @@ contract Platform {
         _;
     }
 
+    /*Ensure caller is a verified seller*/
+    modifier isOrganiser() {
+        require(accountContract.viewAccountState(msg.sender) == accountContract.getVerifiedStatus(),"Your are not a verified seller");
+        _;
+    }
+
+    // list Event on Platform
+    function listEvent(string memory title,
+        string memory venue,
```

Comment:
Perhaps can remove memory? I don't think parameters need this keyword.

Reply:
from solidity:
proj/Platform.sol:36:9: TypeError: Data location must be "memory" for parameter in function, but none was given.
        string venue,
        ^----------^
        
have the above error if remove memory